docs/getting-started-on-snowplow-open-source/quick-start-aws/index.md

```diff
@@ -88,95 +96,78 @@ terraform plan
+If you choose Postgres as destination, there are no additional steps. Respective variables need to be filled according to the desired setup. Necessary resources like a PostgreSQL RDS instance, database, table, user will be created by the Terraform modules.
 
-**Step 2 (Snowflake only): Run Snowflake terraform module**
+It is strongly recommended to:
 
-It is possible to use Snowflake as the destination in AWS pipelines. However necessary resources need to be created in Snowflake before starting the pipeline.
+1. Update the `postgres_db_password` to a new value that only **you** know
+2. Set the `postgres_db_ip_allowlist` to a list of CIDR addresses that will need to access the database - this can be systems like BI Tools, or your home address so you can query the database from your laptop
 
-For this purpose, the [Snowflake Terraform module](https://github.com/snowplow/quickstart-examples/tree/main/terraform/aws/snowflake) has been created. This module creates resources including, but not limited to, Snowflake database, table, user, and role. These resources are needed by the Snowflake Loader to operate correctly.
+##### Snowflake, Redshift & Databricks
 
-#### Prerequisites
+If you choose Snowflake, Redshift or Databricks as a destination, there is one more additional step. Pipeline Terraform module doesn't create necessary resources for these destinations. Unlike the Postgres resources it only deploys the loader itself. Therefore destination resources need to be created *before* deploying the pipeline.
 
-Authentication for the service user is required for the Snowflake Terraform provider - [follow this tutorial](https://quickstarts.snowflake.com/guide/terraforming_snowflake/index.html) to obtain Snowflake connection details:
+The setup instructions for each destination can be found on the loader module page:
 
-| Parameter          | Description                                    |
-|--------------------|------------------------------------------------|
-| account            | The account name.                              |
-| username           | A snowflake user to perform resource creation. |
-| region             | Region for the snowflake deployment.           |
-| role               | Needs to be ACCOUNTADMIN or similar.           |
-| private_key_path | Path the private key.                          |
+* Snowflake: https://github.com/snowplow-devops/terraform-aws-snowflake-loader-ec2#usage
```

Comment:
Not sure I like that the instructions are in a separate repo. Could we move them here?

I also have a few questions about those instructions, e.g. do we need to create the Snowflake schema? I think the latest RDB loader creates that by itself.